pp.c

```diff
@@ -6517,10 +6517,10 @@ PP_wrapped(pp_reverse, 0, 1)
             SP = MARK + 1;
```

Comment:
"scaler" -> "scalar" in the commit message

Reply:
Thanks, Tony